app/lib/actions/katello/capsule_content/sync.rb

```diff
@@ -39,7 +39,7 @@ def plan(smart_proxy, options = {})
                                              environment:  environment,
                                              repository: repository)
           sequence do
-            plan_action(Actions::Pulp::Orchestration::Repository::RefreshRepos, smart_proxy, refresh_options)
+            plan_action(Actions::Pulp::Orchestration::Repository::RefreshRepos, smart_proxy, refresh_options) unless smart_proxy.pulp3_enabled?
```

Comment:
i think this needs to check if pulp2 is enabled, not 'unless pulp3 is enabled'.  specifically ' has_feature?( PULP_NODE_FEATURE)

Reply:
i guess in practice what you have is fine (since we'll never have a mixed capsule), but i think its more correct (and easier to work with in dev to check for the pulp_node feature